src/DiabloInterface/Gui/MainWindow.cs

```diff
@@ -177,6 +178,35 @@ void DebugMenuItemOnClick(object sender, EventArgs e)
             debugWindow.Show();
         }
 
+        void DifficultyNormalToolStripMenuItemOnClick(object sender, EventArgs e)
+        {
+            UnheckDifficultyMenuItems();
```

Comment:
I feel unsure about the context menu having the option to set the difficulty (overriding the ingame-difficulty), while the character class is not overwritten. I ll think about it more, maybe there is a better solution?